routes/events.js

```diff
@@ -0,0 +1,4 @@
+// import { Router } from 'express';
+// import eventController from '../controllers/events';
+
+// const router = Router();
```

Comment:
Please update or remove

Reply:
Update before resolve.
Please add a comment and I will resolve it by myself if I think the updates are verified.